stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -514,7 +515,44 @@ private static void enrichWithInboundRequestInfo(BStruct inboundRequestStruct,
         }
     }
 
-    public static void enrichConnectionInfo(BStruct connection, HTTPCarbonMessage cMsg) {
+    /**
+     * Populate connection information.
+     *
+     * @param connection Represent an connection struct
+     * @param cMsg Represent carbon message.
+     * @param httpResource Represent carbon Http Resource.
+     */
+    public static void enrichConnectionInfo(BStruct connection, HTTPCarbonMessage cMsg, HttpResource httpResource) {
```

Comment:
Wouldn't it be much readable if we could rename cMsg to inboundMsg or something similar?